roles/provision-keycloak-apb/tasks/provision-keycloak.yml

```diff
@@ -127,4 +127,10 @@
   register: copy_jar
   retries: 60
   until: copy_jar.rc == 0
-  delay: 5
\ No newline at end of file
+  delay: 5
+
+- name: Encode admin user credentials
```

Comment:
So this should probably only be done if it is not a shared service

Reply:
This is not shared. Provision keycloak task is done only for original instance